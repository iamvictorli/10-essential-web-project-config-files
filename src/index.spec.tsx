import React from 'react'

import { render } from '@testing-library/react'
import { expect, test } from 'vitest'

test('math', () => {
  expect(1 + 1).toBe(2)
})

test('render', () => {
  const { getByText, container } = render(<div>hello world</div>)
  expect(getByText('hello world')).toBeTruthy()
  expect(container).toMatchSnapshot()
})
