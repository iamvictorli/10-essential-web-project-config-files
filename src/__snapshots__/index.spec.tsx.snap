// Vitest Snapshot v1

exports[`render 1`] = `
<div>
  <div>
    hello world
  </div>
</div>
`;
